mesheryctl/pkg/utils/platform.go

```diff
@@ -781,3 +781,110 @@ func SetKubeConfig() {
 		KubeConfig = filepath.Join(usr.HomeDir, ".kube", "config")
 	}
 }
+
+func ForceCleanupCluster() error {
+	client, err := meshkitkube.New([]byte(""))
+	if err != nil {
+		return errors.Wrap(err, "failed to create new client")
+	}
+
+	deletePolicy := metav1.DeletePropagationForeground
```

Comment:
This takes cares of deleting resources with dependencies. So, we are good.

Reply:
ooh, now I understood your what you were asking in the meeting. 😅